std/node/_stream/end-of-stream.ts

```diff
@@ -0,0 +1,247 @@
+import { once } from "../_utils.ts";
+import type Readable from "./readable.ts";
+import type Writable from "./writable.ts";
+import {
+  ERR_INVALID_ARG_TYPE,
+  ERR_STREAM_PREMATURE_CLOSE,
+  NodeErrorAbstraction,
+} from "../_errors.ts";
+
+type Stream = Readable | Writable;
+
+function isRequest(stream: Stream) {
+  // deno-lint-ignore ban-ts-comment
+  //@ts-ignore
+  return stream.setHeader && typeof stream.abort === "function";
+}
+
+function isReadable(stream: Stream) {
+  // deno-lint-ignore ban-ts-comment
+  //@ts-ignore
```

Comment:
Why do we have to ignore this?

Reply:
I'm gonna refactor that whole file, seeing that I had to refactor it in the Duplex PR anyways. All that functions are gonna probably be receiving any.